channels/alpha

```diff
@@ -20,8 +20,10 @@ spec:
     requiredVersion: 1.4.2
   kopsVersions:
   - range: ">=1.5.0-alpha1"
-    recommendedVersion: 1.5.0-beta1
-    #requiredVersion: 1.5.0-beta1
+    recommendedVersion: 1.5.0-beta2
+    #requiredVersion: 1.5.0-beta2
+    kubernetesVersion: 1.5.2
   - range: "<1.5.0"
     recommendedVersion: 1.4.4
     #requiredVersion: 1.4.4
+    kubernetesVersion: 1.4.8
```

Comment:
They are in separate blocks.  The logic is that if you're using kops 1.4, you should be running kubernetes 1.4.8.  If you're using kops 1.5, you should be running kubernetes 1.5.2.